doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,77 @@
 Changelog
 =========
 
+.. _changelog-v2-9-0:
+
+v2.9.0
+------
+Highlights
+
+TODO: add highlights
```

Comment:
@ESMValGroup/esmvaltool-coreteam Please comment here if you have suggestions for highlights.

Reply:
I suggest #3155 which added a diagnostic that provides a high-level interface to [seaborn](https://seaborn.pydata.org/), a Python data visualization library based on [matplotlib](https://matplotlib.org/).

Reply:
I suggest (in addition to #3155) #3021 which is a new diagnostic about Arctic-midlatitude teleconnetions.

Reply:
Thanks for the ideas! @remi-kazeroni Would you be able to provide a bit of text describing #3155 (since @schlunma is on holiday) and @egalytska would you be able to do the same for #3021? You can leave it here as a comment or directly commit it to this branch.

Reply:
For #3155:

A new [diagnostic](https://docs.esmvaltool.org/en/latest/api/esmvaltool.diag_scripts.seaborn_diag.html#api-esmvaltool-diag-scripts-seaborn-diag) has been added to provide a high-level interface to [seaborn](https://seaborn.pydata.org/), a Python data visualization library based on [matplotlib](https://matplotlib.org/). See [recipe documentation](https://docs.esmvaltool.org/en/latest/recipes/recipe_seaborn.html) for more information.

Reply:
Both suggestions have been added to the text.